course/templates/course/_course_menu.html

```diff
@@ -7,6 +7,12 @@
 
 {% prepare_course_menu %}
 
+<li>
+	<a href="#course-content" class="skip-link page-skip-link" aria-label="{% trans "Course navigation" %}">
```

Comment:
Few thoughts that came into my mind (i.e. unimportant and very minor)

Should the `aria-label` be in the [parent nav element](https://github.com/apluslms/a-plus/blob/3368af1e4ca937d601730c925be9d29aa97e5a96/course/templates/course/course_base.html#L21)? For example, should we consider the advert box to be part of "course navigation" or not (I think it's part of the left panel, but I'm not 100% if all of that is course navigation)? The main navigation now has it in order where link comes before the aria-label, so is that better?

I feel that I'm unsure how to read `aria-label` in the source. Like, should I think that `aria-label` tells "this element contains thing X" or "thing X starts from here"?

If this `li` would be in `course_base.html` just before the include statement, then the link and destination would be in the same file. Though, everything else is in this partial file, so I'm not sure which is better. Additionally, it does not need to be under the `ul` element either, unless that is beneficial for the style (i.e. it would be fine to have a dedicated entry in the `nav` before `ul` just for this).

Reply:
Oh dear. This is the same copy/paste error I fixed yesterday, just in the other component. Will fix it next.

Reply:
Fixed this now @raphendyr